website/docs/tutorials/app-demo-tutorial.md

```diff
@@ -0,0 +1,566 @@
+```
+
+
+We're now ready to deploy!
+
+
+## Deployment
+
+### Deploy container
+
+From a terminal, simply execute:
+
+```sh
+docker compose up
+```
+
+The application container image will be built and run. If successful, you should see the following:
+
+```
+Attaching to nextjs-unleash-demo-nextjs_material-1
+nextjs-unleash-demo-nextjs_material-1  | 
+nextjs-unleash-demo-nextjs_material-1  | > @minimal/material-kit-nextjs@3.0.0 start
+nextjs-unleash-demo-nextjs_material-1  | > next start
+nextjs-unleash-demo-nextjs_material-1  | 
+nextjs-unleash-demo-nextjs_material-1  | ready - started server on 0.0.0.0:3000, url: http://localhost:3000
+```
+
+
+### Validate and Login
+
+Open `http://localhost:3000` in a browser. Click the *Live Preview* button and sign in with the displayed credentials:  
+
+```
+user: demo@minimals.cc
+password : demo1234
+```
+
+A dashboard should load. We're ready to go!
+
+:::tip Errors?
+If you see an Internal Server Error or a different response from the browser, something has gone wrong. Double check the console output as well as the environment variables in the docker compose file.
+:::
+
+# Feature Flags In Action
+
+Although totally transparent to the end user visiting the website, we need to distinguish between client (frontend) and server (backend) feature flags to gain an understanding of how they work and operate in code.
+
+The app is setup for three client side toggles, and one server side toggle. In this section we'll review the UX impact of enabling/disabling each of the flags in turn.
+
+To prepare, open the Unleash project and the application UI side-by-side. 
+As we are using the *Development* environment, these are the UI toggles for the feature flags we will be using.
+
+![Project page](/img/tutorial-appdemo-projectpage.png)
+
+## Client-side Feature Flags
+
+### Banking feature
+
+Enable the toggle for feature `demo-bankingDemo`. Notice how now a new *Banking* feature has been added to the sidebar underneath Analytics!
+
+![demo-bankingDemo](/img/tutorial-appdemo-demo-bankingDemo.png)
+
+
+Click on `Banking` in the navigation menu. Notice how the feature is populated. Disable the flag - this now switches to a *Coming Soon* message.
+
+***tip Use Case
+This shows how a feature flag can be used to expose a given feature to a specific group of users - for example, customers that have a matching profile role, users opted in for early feature access - or for internal development use, e.g. a QA team.  
+It also showcases how quickly and effortlessly a feature can be rolled back if it does not behave as expected.
+***
+
+### Special Customer
+
+In the UI, navigate to *Management -> E-Commerce -> Shop*. Enable the toggle for feature `demo-specialCustomer` . 
+Notice how cards are updated with *Special Access Granted* and a *Premium Customer* note appears at the top. <!-- Add Screenshot -->
+
+![demo-specialCustomer](/img/tutorial-appdemo-demo-specialCustomer.png)
+
+***tip Use Case
```

Comment:
The same goes for these asterisks and the next ones: they should probably be colons.